frontend/src/app/supportAdmin/PendingOrganizations/PendingOrganizationsContainer.test.tsx

```diff
@@ -589,9 +589,6 @@ describe("PendingOrganizationsContainer", () => {
       await waitForElementToBeRemoved(
         screen.queryByText("Verify organization")
       );
-      await waitForElementToBeRemoved(
```

Comment:
deleting this check because it was causing intermittent test failures and is redundant anyways since the "Verify Organization" title on the modal should suffice for the request going through